### DISCUSSION TITLE
Automatic Renaming of Files

### DISCUSSION THREAD

**Discussion by MatchstickWorks** [jmbannon/ytdl-sub#733] [Q&A] [answered]
Are there any regex options/methods available to rename files that are downloaded? For instance, a [playlist that I'm currently trying to get working](https://www.youtube.com/playlist?list=PLjMDIlamRhokw_f_ptzWkhOawBA80c5VP) has changed their naming scheme over the course of the series. It would be great to be able to automatically remove some of the fluff at the beginning of some of the titles and end of titles.

**Comment by jmbannon** [comment, marked as answer]
Not at this time, but there is underlying infra in the code to do this kind of an update. It's on my radar.
Related tasks:
https://github.com/jmbannon/ytdl-sub/issues/536
https://github.com/jmbannon/ytdl-sub/issues/502 (this one requires being able to do metadata updates)

**Reply by MatchstickWorks** [reply]
Great news, thanks!
